code/modules/SCP/SCPs/SCP-457.dm

```diff
@@ -0,0 +1,228 @@
+			to_chat(src, "<span class='warning'>You can't attack yet.</span>")
+			return
+		else
+			if(prob(35))
+				visible_message(SPAN_WARNING("[src] begins to initiate agony in [A]!"))
+				if(do_after(src, 1 SECOND, H))
+					H.Weaken(10)
+					H.visible_message("<span class='danger'>[src] claws at [H], the flame sending them to the floor!</span>")
+					to_chat(H, "<span class='userdanger'>IT HURTS!!!</span>")
+					health += 5
+					aflame_cooldown = world.time + aflame_cooldown_time
+				return
+			else
+				visible_message(SPAN_WARNING("[src] raises their firey arms and begins to attack [A]!"))
+				if(do_after(src, 3 SECONDS, H))
+					H.fire_stacks += 1
+					H.IgniteMob()
+					health += 15
+					aflame_cooldown = world.time + aflame_cooldown_time
+					visible_message("<span class='danger'>[src] claws at [A] setting them alight!</span>")
+					to_chat(H, "<span class='userdanger'>Oh god, oh god. OH GOD! IT HURTS! PLEASE!</span>")
+					return
+	if(istype(A, /obj/machinery/door))
+		OpenDoor(A)
+		return
+	if(istype(A, /obj/structure/grille))
+		playsound(get_turf(A), 'sound/effects/grillehit.ogg', 50, 1)
+		qdel(A)
+		return
+	if(istype(A, /obj/structure/inflatable))
+		var/obj/structure/inflatable/W = A
+		W.deflate(violent=1)
+	return
+
+/mob/living/scp_457/proc/OpenDoor(obj/machinery/door/A)
+	if(door_cooldown > world.time)
+		return
+
+	if(!istype(A))
+		return
+
+	if(!A.density)
+		return
+
+	if(!A.Adjacent(src))
+		to_chat(src, "<span class='warning'>\The [A] is too far away.</span>")
+		return
+
+	var/open_time = 4 SECONDS
+	if(istype(A, /obj/machinery/door/blast))
+		if(get_area(A) == spawn_area)
+			to_chat(src, "<span class='warning'>This blast door is too thermally protected, you cannot melt through it.</span>")
+			return
+		open_time = 15 SECONDS
+
+	if(istype(A, /obj/machinery/door/airlock))
+		var/obj/machinery/door/airlock/AR = A
+		if(AR.locked)
+			open_time += 3 SECONDS
+		if(AR.welded)
+			open_time += 2 SECONDS
+		if(AR.secured_wires)
+			open_time += 4 SECONDS
+
+	A.visible_message(SPAN_WARNING("\The [src] begins to melt the control mechanisms on \the [A]!"))
+	playsound(get_turf(A), 'sound/machines/airlock_creaking.ogg', 35, 1)
+	door_cooldown = world.time + open_time // To avoid sound spam
+	if(!do_after(src, open_time, A))
+		return
+
+	if(istype(A, /obj/machinery/door/blast))
+		var/obj/machinery/door/blast/DB = A
+		DB.visible_message(SPAN_DANGER("\The [src] forcefully opens \the [DB]!"))
+		DB.force_open()
+		return
+
+	if(istype(A, /obj/machinery/door/airlock))
+		var/obj/machinery/door/airlock/AR = A
+		AR.unlock(TRUE) // No more bolting in the SCPs and calling it a day
+		AR.welded = FALSE
+	A.stat |= BROKEN
+	var/check = A.open(1)
+	src.visible_message("\The [src] melts \the [A]'s controls[check ? ", and rips it open!" : ", and breaks it!"]")
+
+/datum/reagent/water/touch_mob(var/mob/living/scp_457/M)
+	M.adjustToxLoss(30)
+	to_chat(M, "<span class='userdanger'>FUEL LESSENS, MAKE THEM PAY...")
```

Comment:
Did you test this? it looks too stupid to work.

Reply:
Also, adjust tox loss?

Reply:
I believe that this code makes all the water in fire extinguishers extremely deadly for anyone.

Reply:
made us all grays from vg

Reply:
fixed